packages/labs/analyzer/CHANGELOG.md

```diff
@@ -32,7 +32,7 @@
 
 - [#2804](https://github.com/lit/lit/pull/2804) [`e0517ccf`](https://github.com/lit/lit/commit/e0517ccf79d983a8d6ec53969f29e130830fe3e8) - Read property options from decorated properties
 
-- [#2896](https://github.com/lit/lit/pull/2896) [`3bd330f3`](https://github.com/lit/lit/commit/3bd330f3db4c2f618181b8602563db3ab879f33d) - Add utilties for getting LitElement declarations
```

Comment:
Some projects don't like changing changelogs... happy to drop this or anything else.